js/components/statistics/index.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component} from 'react';
+
+// imports for table
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+
+// imports for charts
+import Chart from '../chart';
+import {repositories} from '../../mocks';
+
+import statistics_style from './statistics.scss';
+
+export default class Statistics extends Component {
+  constructor(...args) {
+    super(...args)
+    this.state = {
+      chartCollections: []
+    }
+    this._onRowSelection = this._onRowSelection.bind(this);
+  }
+  
+  _onRowSelection(rows: Array<number>) {
```

Comment:
Array<number> - it's strange construction

Reply:
removed